src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
```

Comment:
Per task:

> All instances of the Animal class must have properties health and a name. Health is 100 by default. 
> Create a Herbivore class. Herbivore has a method of hide, which changes the hidden property of the beast and helps to hide from carnivores.

Probably some of these properties should be in another class

Reply:
yes sure, no problem